content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Installing External Istiod [experimental]
+description: Install external Istiod and remote config cluster.
+weight: 46
+keywords: [external,istiod,remote,config]
+aliases:
+    - /docs/setup/kubernetes/additional-setup/external-istiod/
```

Comment:
why do you need an alias? My understanding was that aliases were mainly for redirects to old pages that were moved/replaced?

Reply:
probably not needed... removed.